setup.py

```diff
@@ -1,7 +1,7 @@
-from setuptools import setup, find_packages
+from setuptools import find_packages, setup
 
 requires = [
-    'django',
+    'Django>=1.11',
```

Comment:
+0. I think some users can use a part of this backend library if this version limitation is not exist (if they lucky). However I don’t oppose version constraints.